src/vite.config.js

```diff
@@ -0,0 +1,14 @@
+const { resolve } = require('path')
+
+module.exports = {
+  build: {
+    rollupOptions: {
+      input: {
+        main: resolve(__dirname, 'index.html'),
+        buddies: resolve(__dirname, 'code-buddys.html'),
+        teams: resolve(__dirname, 'teams.html'),
+        energy: resolve(__dirname, 'energy.html'),
+      },
+    },
+  },
+}
```

Comment:
This is for the build script to add the other pages, we'll talk about that in detail in class